docs/pages/api/introduction.mdx

```diff
@@ -4,24 +4,43 @@ description: Introduction to the Teleport gRPC API.
 layout: tocless-doc
 ---
 
-The Teleport Auth API provides a gRPC API for remotely interacting with a Teleport Auth server.
-
-Teleport has a public [Go client](https://pkg.go.dev/github.com/gravitational/teleport/api/client)
-to programatically interact with the API. [tsh and tctl](../reference/cli.mdx) use the same API.
+The Teleport Auth Service provides a gRPC API for remotely interacting with
+your cluster.
 
 ## Go client
 
+Teleport has a public [Go
+client](https://pkg.go.dev/github.com/gravitational/teleport/api/client) to
+programatically interact with the API. [tsh and tctl](../reference/cli.mdx) use
+the same API.
+
 Here is what you can do with the Go Client:
- - Integrating with external tools, which we have already done 
-   for [several tools](../access-controls/access-request-plugins/index.mdx),
-   such as Slack, Jira, and Mattermost.
- - Writing a program/bot to manage Access Requests automatically, based on your use case. One idea
-   is to allow/deny developer requests based on their currently assigned tasks.
- - Performing CRUD actions on resources, such as `roles`, `auth connectors`, and `provisioning tokens`.
+
+ - Integrate with external tools, e.g., to write an [Access Request
+   plugin](../access-controls/access-request-plugins/index.mdx). Teleport
+   maintains Access Request plugins for tools like Slack, Jira, and Mattermost.
+ - Writing a program/bot to manage Access Requests automatically, based on your
```

Comment:
👍 
I've also deleted this particular line (from the original page), since we don't recommend approving/denying Access Requests automatically (and recommend displaying links to the Teleport Web UI to do this).